frontend/src/components/kl-add-item.js

```diff
@@ -28,7 +28,18 @@ export default {
             }).catch(err => {
                 if (err.status === 400 && err.body.code === 1) {
                     let renderedErrors = renderValidationErrors(err.body.errors);
-                    renderedErrors.forEach(error => toastr.error(this.$t('errors.' + error.id, error.props)));
+                    renderedErrors.forEach(error => {
+                        if (error.id === 'mustBeLessThan')
+                            toastr.error(this.$t('errors.' + error.id, error.props));
+                        else {
+                            let res;
+                            if (error.count === 1)
+                                res = 1;
```

Comment:
I don't know, this just doesn't seem right. So much hassle for such trivial thing. It should look like so:
```
toastr.error(translate('errors.' + error.id, error.props));
```
This contains all the necessary stuff - it says that the error should be displayed with toastr, and that it should be translated using certain tranlsation id and certain error data. That is totally enough info to compose proper error message.
All of the nuances of plural/whatever should be handled in that `translate` function. If it is not possible to do so straight with $t or $tc, then we need a separate function. The thing is that component shouldn't care about all that translation details. All it cares about is that there was an error, and it should be displayed in translated manner with toastr.